sherpa/astro/io/tests/test_io.py

```diff
@@ -16,41 +16,43 @@
 #  with this program; if not, write to the Free Software Foundation, Inc.,
 #  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 #
+import pytest
 
-from sherpa.utils.testing import SherpaTestCase, requires_data, \
-    requires_fits, requires_xspec
+import numpy as np
+from sherpa.utils.testing import requires_data, requires_fits, requires_xspec
 from sherpa.astro import ui
 
 from tempfile import NamedTemporaryFile
 
+@pytest.fixture
+def setup(make_data_path):
```

Comment:
`sherpa/conftest.py` has `clean_astro_ui`. Can this be used here instead (it's only a simple function, but there is a little bit of subtlety that can rarely bite with the XSPEC settings)?

Reply:
Although it doesn't look like the fixture is actually being used in any of the tests.

Reply:
I converted the previous tests we had it, but I don't think it's needed, so I'll remove it. (In fact, I think it's good if the tests fail because they *should* not depend on the state. If they do, we need to find out why.)